torch/_dynamo/side_effects.py

```diff
@@ -574,7 +592,12 @@ def codegen_save_tempvars(self, cg: PyCodegen):
                     var.source = LocalCellSource(var.local_name)
             elif isinstance(var.mutation_type, AttributeMutationNew):
                 if isinstance(var, variables.AutogradFunctionContextVariable):
-                    unimplemented("AutogradFunctionContextVariable escaped")
+                    unimplemented_v2(
+                        gb_type="AutogradFunctionContextVariable escaped Dynamo-traced region",
```

Comment:
This sounds more like... "we cannot reconstruct a torch.autograd.Function's ctx object"